docs/sources/flow/reference/components/prometheus.integration.redis.md

```diff
@@ -0,0 +1,104 @@
+---
+# NOTE(rfratto): the title below has zero-width spaces injected into it to
+# prevent it from overflowing the sidebar on the rendered site. Be careful when
+# modifying this section to retain the spaces.
+#
+# Ideally, in the future, we can fix the overflow issue with css rather than
+# injecting special characters.
+
+title: prometheus.​integration.redis
+---
+
+# prometheus.integration.redis
+The `prometheus.integration.redis` component embeds
+[redis_exporter](github.com/oliver006/redis_exporter) for collecting metrics from a redis database.
```

Comment:
The link isn't working for me in the preview.  Might need to include http?  https://github.com/oliver006/redis_exporter